### PR TITLE
unreal plugin snippets

### PR DESCRIPTION
**Pull request by tustanivsky**
Adds documentation with examples for the following Unreal Engine plugin features:
- Attachments
- Scopes

Also, UE wizard was updated with the CRC configuration guide.

### CODE REVIEW

File: src/wizard/unreal/index.md [hunk 1]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
 
-<Alert level="warning" title="Warning">
+## Crash Report Client
 
 For Windows and Mac [Crash Report Client](/platforms/unreal/setup-crashreport/) provided along with Unreal Engine has to be configured in order to capture errors automatically.
 
-</Alert>
+### Include the UE4 Crash Reporter
+
+You can add the _Crash Report Client_ to your game in your _Project Settings_.
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:34]: Is Project Settings a page in the configuration editor or some other UI element below the page level?

File: src/wizard/unreal/index.md [hunk 2]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+
+The option is located under _Project > Packaging_ menu, then select _show advanced_ followed by
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:36]:
```suggestion
The option is located under **Project > Packaging**; select "show advanced" followed by
```

File: src/wizard/unreal/index.md [hunk 3]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+checking the box for: `Include Crash Reporter`.
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:37]:
```suggestion
checking the box for "Include Crash Reporter".
```

File: src/wizard/unreal/index.md [hunk 4]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+### Debug information
+
+To get the most out of Sentry, crash reports are required to include debug information.
+In order for Sentry to be able to process the crash report and translate
+memory addresses to meaningful information like function names, module names
+and line numbers, the crash itself must include debug information and also, symbols need
+to be uploaded to Sentry.
+
+The option is also located under _Project > Packaging_ menu, then select _show advanced_ followed by
+checking the box for: `Include Debug Files`.
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:48]:
```suggestion
checking the box for "Include Debug Files".
```

File: src/wizard/unreal/index.md [hunk 5]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+
+### Configure the Crash Reporter Endpoint
+
+Now that the _Crash Reporter_ and _Debug Files_ are included, UE4 needs to know where to send the
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:52]:
```suggestion
Now that the crash reporter and debug files are included, UE4 needs to know where to send the
```

**Comment by imatwawana** [comment]
> @imatwawana Changes in the wizard file mainly duplicate this [guide](https://docs.sentry.io/platforms/unreal/setup-crashreport/). It provides a bit more context and some screenshots, so that should help you distinguish different elements. Please let me know whether it was helpful.

Thank you! I'll take another look later today.

File: src/wizard/unreal/index.md (hunk 1 above)
Comment (tustanivsky) [reply src/wizard/unreal/index.md:34]: Here `Project Settings` is an Unreal Engine editor window used for project/plugin configuration.

File: src/wizard/unreal/index.md [hunk 6]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+crash. For that, we add the Sentry _Unreal Engine Endpoint_ from the _Client Keys_ settings page to game's configuration file. This will
+include which project within Sentry you want to see the crashes arriving in real time.
```
[the remark this answers is not in the document]
Comment (tustanivsky) [reply src/wizard/unreal/index.md:54]: Agree, let's proceed with the version you've suggested.

File: src/wizard/unreal/index.md [hunk 7]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
 ___PUBLIC_DSN___
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:25]: I can't access these lines from here:

- Line 10, can you change `Plugins` to "Plugins"
- Line 12, put Settings > Plugins > Code Plugins in bold (not the word _menu_ though)
- Line 22, put change Project Settings to just Settings and put that nav path in bold and not code font
-

File: src/wizard/unreal/index.md [hunk 8]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
-<Alert level="warning" title="Warning">
+## Crash Report Client
 
-For Windows and Mac [Crash Report Client](/platforms/unreal/setup-crashreport/) provided along with Unreal Engine has to be configured in order to capture errors automatically.
+For Windows and Mac, [Crash Report Client](/platforms/unreal/setup-crashreport/) provided along with Unreal Engine has to be configured in order to capture errors automatically.
 
-</Alert>
+### Include the UE4 Crash Reporter
+
+You can add the _Crash Report Client_ to your game in your _Project Settings_.
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:34]:
```suggestion
You can add the crash report client to your game in **Project Settings**.
```

File: src/wizard/unreal/index.md [hunk 9]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+
+The option is located under _Project > Packaging_ menu, then select _show advanced_ followed by
+checking the box for: `Include Crash Reporter`.
+
+### Debug Information
+
+To get the most out of Sentry, crash reports must include debug information.
+In order for Sentry to be able to process the crash report and translate
+memory addresses to meaningful information like function names, module names
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:43]:
```suggestion
memory addresses to meaningful information like function names, module names,
```

File: src/wizard/unreal/index.md [hunk 10]
```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
...
+and line numbers, the crash itself must include debug information and symbols need
```
Comment (imatwawana) [inline src/wizard/unreal/index.md:44]:
```suggestion
and line numbers, the crash itself must include debug information. In addition, symbols need
```

**Review by imatwawana — APPROVED** [review]
Okay I've added a few more edits, but the other suggestions I made for the wizard file are correct based on the screenshots you referred me to.

**Review by bruno-garcia — APPROVED** [review]
Thanks Ivan

**Pull request merged**